README.md

```diff
@@ -29,13 +33,19 @@ compile 'com.bandwidth.sdk:messaging:(put desired version here)'
 public class MyAwesomeBandwidthMessagingApp{
     
     public static void main(String[] args){
+        
+        // Create an instance of the Messaging Client
         MessagingClient client = new MessagingClient(
                 "u-aeawj73oafil", // your UserID
                 "t-ayu44kfjhbf", // your api token
                 "soykuhkfalkjdf" // your api secret
-        );
-        String myUploadedMedia = client.uploadMedia("~/path/to/file.jpg","media_file_name.jpg")
-        Message message = client.sendMessage(SendMessageRequest.builder()
+        );        
+        // Upload a local file that can be used for MMS 
+        String myUploadedMedia = client.uploadMedia("/path/to/file.jpg","media_file_name.jpg");
```

Comment:
Since you are uploading your own media, can you add a comment (either here on where we use a public url) that you can also use any publicly accessible media?